### ISSUE TITLE
[Feature] Stop production

### ISSUE THREAD

**Issue by Spikeone** [nicymike/serflings#4]
It'd be nice to have a feature in order to stop production of a building.

**Comment by ShpakMA** [comment]
You can stop all _boat yards_, _toolmakers_, _steel workers_ and _goldsmith_ by set supply to zero in UI. I test it both on Settlers and Serflings. 
 
You may see this practice at set of saves from Issue #107

**Comment by nicymike** [comment]
I think the feature is about to stop an individual building like in Settlers 2. Sometimes it is a bit cumbersome to stop production by setting supply. Also there are buildings which have no supply like lumberjack you may want to stop.

**Comment by ShpakMA** [comment]
> You may see this practice at set of saves from Issue #107

SAVE0.DS [SMA10] have stopped production, but we have resources for production. I think, this is better, than individual stop — no matter what boat yard is eat wood until carriers is carry anything to anybody.
 
![Снимок экрана в 2023-06-09 16-26-47](https://github.com/nicymike/serflings/assets/134811226/ef6d06d6-d754-455e-8a9c-efc746725adb)
![Снимок экрана в 2023-06-09 16-27-04](https://github.com/nicymike/serflings/assets/134811226/2dae2793-8547-4633-8ea8-363c942dc75a)

**Comment by nicymike** [comment]
Yes, but you cannot stop buildings that don't need resources like forester, farm, fisherman or woodcutter.

**Comment by ShpakMA** [comment]
> Yes, but you cannot stop buildings that don't need resources like forester, farm, fisherman or woodcutter.

Yes, but why we need stop it? Stopping Boat Yards or Toolmakers is need for preventing wasting resources… 

But I also think: making Settlers II GUI with Settlers I gameplay is very nice idea…

**Comment by nicymike** [comment]
Maybe you temporarily want to stop a forester, fisherman or farmer. I know people who are asking for this.

**Comment by rtrzebinski** [comment]
Stoping (pausing) mines would be useful in case too many mines were built, and resources are congesting roads.
